scripts/check_lint.sh

```diff
@@ -16,3 +16,4 @@ if [ ! $(command -v ${DOCKER_CMD}) ]; then
 fi
 
 ${DOCKER_CMD} run --rm -e GOPROXY=${GOPROXY} -v $(pwd):/opt/workspace -w /opt/workspace golangci/golangci-lint:v1.21 golangci-lint run
+${DOCKER_CMD} run --rm -e GOPROXY=${GOPROXY} -v $(pwd):/opt/tmp -w /opt/tmp golangci/golangci-lint:v1.21  /bin/bash -c "mkdir -p /opt/workspace;cp -r /opt/tmp/. /opt/workspace;cd /opt/workspace/cmd/aries-agentd;cp ../../.golangci.yml .;golangci-lint run"
```

Comment:
Why not just cd into cmd/aries-agentd? Why copy?

Reply:
I need .golangci.yml  inside cmd/aries-agentd

Reply:
done